src/swift/Queue.swift

```diff
@@ -271,8 +271,8 @@ public extension DispatchQueue {
 	private func _syncHelper<T>(
 		fn: (DispatchWorkItem) -> (), 
 		flags: DispatchWorkItemFlags,
-		execute work: @noescape () throws -> T,
-		rescue: ((Swift.Error) throws -> (T))) rethrows -> T
+		execute work: () throws -> T,
+		rescue: @escaping ((Swift.Error) throws -> (T))) rethrows -> T
```

Comment:
I think `rescue` could be `@noescape`. I can't see nothing keeping a reference to it.